tests/components/mealie/test_config_flow.py

```diff
@@ -83,6 +83,40 @@ async def test_flow_errors(
     assert result["type"] is FlowResultType.CREATE_ENTRY
 
 
+@pytest.mark.parametrize(
+    ("version"),
+    [
+        ("v1.0.0beta-5"),
+        ("v1.0.0-RC2"),
+        ("v0.1.0"),
+        ("something"),
```

Comment:
Awesomeversion is able to handle this case natively

![CleanShot 2024-07-10 at 15 16 59@2x](https://github.com/home-assistant/core/assets/195327/7cc7ee3d-e286-4fe9-bd0f-d1d066859256)

Reply:
Oh nevermind, it is not.